src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -1269,7 +1298,7 @@ private function mergeBranchFromStagingArea($id, $bundle) {
     if ($err) {
       $this->writeWarn(pht('MERGE TAG FAILED'),
         pht('Unable to merge tag'));
-      throw new ArcanistUsageException(pht($message));
```

Comment:
Why remove pht call?

Reply:
It is linting error, pht first parameter should be "constant"... pht sprintf... Doesn't make much sense also "translating" error doesn't make much sense...